dev/com.ibm.ws.security.token.ltpa/resources/com/ibm/ws/security/token/ltpa/internal/resources/LTPAMessages.nlsprops

```diff
@@ -72,6 +72,14 @@ LTPA_KEYS_FILE_DOES_NOT_EXIST=CWWKS4112E: The {0} LTPA validation keys file does
 LTPA_KEYS_FILE_DOES_NOT_EXIST.explanation=The validation keys file does not exist.
 LTPA_KEYS_FILE_DOES_NOT_EXIST.useraction=Make sure that the validation keys file exists in the specified location that is specified by the validationKeys element.
 
-LTPA_MONITOR_DIRECTORY_TRUE_AND_FILE_MONITOR_NOT_ENABLED=CWWKS4113W: The monitorDirectory attribute is set to true but the monitor interval is {0}. No dynamic reload occurs when validation keys files are created, modified or deleted in the directory.
-LTPA_MONITOR_DIRECTORY_TRUE_AND_FILE_MONITOR_NOT_ENABLED.explanation= The value for the monitorInterval attribute  is zero or less than zero. The directory cannot be monitored for any create, delete, or modify actions on files with the .keys suffix and dynamic reload cannot occur.
-LTPA_MONITOR_DIRECTORY_TRUE_AND_FILE_MONITOR_NOT_ENABLED.useraction=Set the monitorInterval attribute to an interval that is greater than zero.
\ No newline at end of file
+LTPA_MONITOR_VALIDATION_KEYS_DIR_TRUE_AND_MONITOR_INTERVAL_EQUALS_TO_OR_LESS_THAN_ZERO=CWWKS4113W: The monitorValidationKeysDir attribute is set to true but the monitor interval is {0}. No dynamic reload occurs when LTPA key files are created, modified or deleted in the directory.
+LTPA_MONITOR_VALIDATION_KEYS_DIR_TRUE_AND_MONITOR_INTERVAL_EQUALS_TO_OR_LESS_THAN_ZERO.explanation=The value for the monitorInterval attribute is zero or less than zero. The directory cannot be monitored for any create, delete, or modify actions on files with the .keys suffix and dynamic reload cannot occur.
+LTPA_MONITOR_VALIDATION_KEYS_DIR_TRUE_AND_MONITOR_INTERVAL_EQUALS_TO_OR_LESS_THAN_ZERO.useraction=Set the monitorInterval attribute to an interval that is greater than zero.
+
+LTPA_UPDATE_TRIGGER_DISABLED_AND_MONITOR_VALIDATION_KEYS_DIR_TRUE=CWWKS4114W: The updateTrigger attribute is set to disabled and the monitorValidationKeysDir attribute is set to true. No dynamic reload occurs when LTPA key files are created, modified or deleted in the directory.
+LTPA_UPDATE_TRIGGER_DISABLED_AND_MONITOR_VALIDATION_KEYS_DIR_TRUE.explanation=When the updateTrigger attribute is set to disabled, the directory will not be monitored for any create, delete, or modify actions on files with the .keys suffix and dynamic reload will not occur. Even if the monitorValidationKeysDir attribute is set to true, the attribute will be ignored, and the directory will not be monitored.
```

Comment:
change
will not be monitored
to
is not monitored

change
will not occur.
to
does not occur.

change
will be ignored
to
is ignored

change
will not be monitored.
to
is not monitored.